repl/commands_test.go

```diff
@@ -104,10 +110,6 @@ func TestParseErrors(t *testing.T) {
 			// no assignment
 			commandLine: "%let x: int64",
```

Comment:
Do you want to maybe use an invalid type name rather than one that looks close to right?

Reply:
updated